### ISSUE TITLE
Errors: Unable to find collection for file at content\global\index.json

### ISSUE THREAD

**Issue by BaseMax** [tinacms/tinacms#3274]
![image](https://user-images.githubusercontent.com/2658040/195474611-68afefa9-44e0-4c4d-ba5d-278cd7540830.png)

![image](https://user-images.githubusercontent.com/2658040/195474811-818bec5f-1ad5-470a-b75f-d85a669339b1.png)

## Commands

```

C:\Users\Max\Desktop\tina>npx create-tina-app@latest
√ Which package manager would you like to use? » Yarn
√ What is your project named? ... my-tina-app
√ What starter code would you like to use? » Tailwind Starter
The directory my-tina-app contains files that could conflict:

  .env.example
  .eslintignore
  .eslintrc
  .github/
  .prettierrc
  .tina/
  .vscode/
  assets/
  components/
  content/
  graphql.config.js
  next-env.d.ts
  next.config.js
  NOTICE
  package.json
  pages/
  postcss.config.js
  public/
  README.md
  styles.css
  tailwind.config.js
  tsconfig.json
  yarn.lock

Either try using a new directory name, or remove the files listed above.


C:\Users\Max\Desktop\tina>dir
 Volume in drive C has no label.
 Volume Serial Number is 0E60-C99C

 Directory of C:\Users\Max\Desktop\tina

10/13/2022  02:41 AM    <DIR>          .
10/13/2022  02:40 AM    <DIR>          ..
10/13/2022  02:41 AM    <DIR>          my-tina-app
               0 File(s)              0 bytes
               3 Dir(s)  15,270,227,968 bytes free

C:\Users\Max\Desktop\tina>cd my-tina-app

C:\Users\Max\Desktop\tina\my-tina-app>dir
 Volume in drive C has no label.
 Volume Serial Number is 0E60-C99C

 Directory of C:\Users\Max\Desktop\tina\my-tina-app

10/13/2022  02:41 AM    <DIR>          .
10/13/2022  02:41 AM    <DIR>          ..
10/03/2022  03:20 PM               206 .env.example
10/03/2022  03:20 PM                17 .eslintignore
10/03/2022  03:20 PM               239 .eslintrc
10/13/2022  02:41 AM    <DIR>          .github
10/03/2022  03:20 PM               207 .gitignore
10/03/2022  03:20 PM                 2 .prettierrc
10/13/2022  02:41 AM    <DIR>          .tina
10/13/2022  02:41 AM    <DIR>          .vscode
10/13/2022  02:41 AM    <DIR>          assets
10/13/2022  02:41 AM    <DIR>          components
10/13/2022  02:41 AM    <DIR>          content
10/03/2022  03:20 PM               339 graphql.config.js
10/03/2022  03:20 PM            11,357 LICENSE
10/03/2022  03:20 PM               201 next-env.d.ts
10/03/2022  03:20 PM               186 next.config.js
10/03/2022  03:20 PM               160 NOTICE
10/03/2022  03:20 PM             1,266 package.json
10/13/2022  02:41 AM    <DIR>          pages
10/03/2022  03:20 PM               140 postcss.config.js
10/13/2022  02:41 AM    <DIR>          public
10/03/2022  03:20 PM             2,843 README.md
10/03/2022  03:20 PM                59 styles.css
10/03/2022  03:20 PM             6,368 tailwind.config.js
10/03/2022  03:20 PM               611 tsconfig.json
10/03/2022  03:20 PM           555,098 yarn.lock
              17 File(s)        579,299 bytes
              10 Dir(s)  15,270,031,360 bytes free

C:\Users\Max\Desktop\tina\my-tina-app>npm run dev

> @tinacms/starter@0.1.0 dev
> tinacms dev -c "next dev"

'tinacms' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Max\Desktop\tina\my-tina-app>npm install
npm WARN deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
npm WARN deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm WARN deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
npm WARN deprecated sane@4.1.0: some dependency vulnerabilities fixed, support for node < 10 dropped, and newer ECMAScript syntax/features added
npm WARN deprecated mdast@3.0.0: `mdast` was renamed to `remark`
npm WARN deprecated svgo@1.3.2: This SVGO version is no longer supported. Upgrade to v2.x.x.
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated core-js@1.2.7: core-js@<3.23.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Some versions have web compatibility issues. Please, upgrade your dependencies to the actual version of core-js.
npm WARN deprecated core-js@2.6.12: core-js@<3.23.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Some versions have web compatibility issues. Please, upgrade your dependencies to the actual version of core-js.

added 1967 packages, and audited 1968 packages in 11m

283 packages are looking for funding
  run `npm fund` for details

16 high severity vulnerabilities

To address issues that do not require attention, run:
  npm audit fix

To address all issues possible, run:
  npm audit fix --force

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.

C:\Users\Max\Desktop\tina\my-tina-app>npm run dev

> @tinacms/starter@0.1.0 dev
> tinacms dev -c "next dev"


No format provided for collection page, defaulting to .md
        Typescript types => C:\Users\Max\Desktop\tina\my-tina-app/.tina/__generated__/types.ts
        GraphQL types ====> C:\Users\Max\Desktop\tina\my-tina-app/.tina/__generated__/schema.gql

Visit the GraphQL playground at http://localhost:4001/altair/
or
Enter the CMS at [your-development-url]/admin

ready - started server on 0.0.0.0:3000, url: http://localhost:3000

warn - You have enabled the JIT engine which is currently in preview.
warn - Preview features are not covered by semver, may introduce breaking changes, and can change at any time.
event - compiled client and server successfully in 24.1s (1165 modules)
Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry

wait  - compiling /[filename] (client and server)...
event - compiled client and server successfully in 11.5s (1243 modules)
Unable to find collection for file at content\global\index.json

GraphQL request:10:3
 9 |     fragment LayoutQueryFragment on Query {
10 |   global(relativePath: "index.json") {
   |   ^
11 |     ...GlobalParts
More error context below
Unable to find collection for file at content\global\index.json
Error: Unable to find collection for file at content\global\index.json
    at TinaSchema.getCollectionByFullPath (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\@tinacms\graphql\dist\index.js:2676:15)
    at TinaSchema.getCollectionAndTemplateByFullPath (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\@tinacms\graphql\dist\index.js:2682:31)
    at Database.get (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\@tinacms\graphql\dist\index.js:3861:53)
    at async Resolver.getDocument (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\@tinacms\graphql\dist\index.js:3072:23)
    at async fieldResolver (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\@tinacms\graphql\dist\index.js:3705:39)
    at async Promise.all (index 0)
    at async Object.resolve (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\@tinacms\graphql\dist\index.js:3588:17)
    at async C:\Users\Max\Desktop\tina\my-tina-app\node_modules\@tinacms\cli\dist\index.js:246:24 {
  locations: [ { line: 10, column: 3 } ],
  path: [ 'global' ]
}
error - Error: Unable to fetch, please see our FAQ for more information: https://tina.io/docs/errors/faq/

        Errors:
        Unable to find collection for file at content\global\index.json
    at TinaClient.request (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\tinacms\dist\client.js:74:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async requester (webpack-internal:///./.tina/__generated__/types.ts:454:22)
    at async getStaticProps (webpack-internal:///./pages/[filename].tsx:45:23)
    at async Object.renderToHTML (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\next\dist\server\render.js:390:20)
    at async doRender (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\next\dist\server\base-server.js:642:38)
    at async cacheEntry.responseCache.get.isManualRevalidate.isManualRevalidate (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\next\dist\server\base-server.js:747:28)
    at async C:\Users\Max\Desktop\tina\my-tina-app\node_modules\next\dist\server\response-cache\index.js:80:36 {
  page: '/[filename]'
}
wait  - compiling /_error (client and server)...
event - compiled client and server successfully in 464 ms (1244 modules)
Cannot query field "id" on type "DocumentConnection".

GraphQL request:7:13
6 |           documents {
7 |             id
  |             ^
8 |             sys {
More error context below
Cannot query field "id" on type "DocumentConnection".
GraphQLError [Object]: Cannot query field "id" on type "DocumentConnection".
    at Object.Field (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\graphql\validation\rules\FieldsOnCorrectTypeRule.js:48:31)
    at Object.enter (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\graphql\language\visitor.js:323:29)
    at Object.enter (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\graphql\utilities\TypeInfo.js:370:25)
    at visit (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\graphql\language\visitor.js:243:26)
    at validate (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\graphql\validation\validate.js:69:24)
    at graphqlImpl (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\graphql\graphql.js:99:49)
    at C:\Users\Max\Desktop\tina\my-tina-app\node_modules\graphql\graphql.js:28:31
    at new Promise (<anonymous>)
    at graphql (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\graphql\graphql.js:26:10)
    at Object.resolve (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\@tinacms\graphql\dist\index.js:3588:51) {
  locations: [ { line: 7, column: 13 } ]
}
Cannot query field "sys" on type "DocumentConnection".

GraphQL request:8:13
7 |             id
8 |             sys {
  |             ^
9 |               filename
More error context below
Cannot query field "sys" on type "DocumentConnection".
GraphQLError [Object]: Cannot query field "sys" on type "DocumentConnection".
    at Object.Field (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\graphql\validation\rules\FieldsOnCorrectTypeRule.js:48:31)
    at Object.enter (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\graphql\language\visitor.js:323:29)
    at Object.enter (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\graphql\utilities\TypeInfo.js:370:25)
    at visit (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\graphql\language\visitor.js:243:26)
    at validate (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\graphql\validation\validate.js:69:24)
    at graphqlImpl (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\graphql\graphql.js:99:49)
    at C:\Users\Max\Desktop\tina\my-tina-app\node_modules\graphql\graphql.js:28:31
    at new Promise (<anonymous>)
    at graphql (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\graphql\graphql.js:26:10)
    at Object.resolve (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\@tinacms\graphql\dist\index.js:3588:51) {
  locations: [ { line: 8, column: 13 } ]
}
Unable to find collection for file at content\global\index.json

GraphQL request:10:3
 9 |     fragment LayoutQueryFragment on Query {
10 |   global(relativePath: "index.json") {
   |   ^
11 |     ...GlobalParts
More error context below
Unable to find collection for file at content\global\index.json
Error: Unable to find collection for file at content\global\index.json
    at TinaSchema.getCollectionByFullPath (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\@tinacms\graphql\dist\index.js:2676:15)
    at TinaSchema.getCollectionAndTemplateByFullPath (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\@tinacms\graphql\dist\index.js:2682:31)
    at Database.get (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\@tinacms\graphql\dist\index.js:3861:53)
    at async Resolver.getDocument (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\@tinacms\graphql\dist\index.js:3072:23)
    at async fieldResolver (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\@tinacms\graphql\dist\index.js:3705:39)
    at async Promise.all (index 0)
    at async Object.resolve (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\@tinacms\graphql\dist\index.js:3588:17)
    at async C:\Users\Max\Desktop\tina\my-tina-app\node_modules\@tinacms\cli\dist\index.js:246:24 {
  locations: [ { line: 10, column: 3 } ],
  path: [ 'global' ]
}
error - Error: Unable to fetch, please see our FAQ for more information: https://tina.io/docs/errors/faq/

        Errors:
        Unable to find collection for file at content\global\index.json
    at TinaClient.request (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\tinacms\dist\client.js:74:15)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async requester (webpack-internal:///./.tina/__generated__/types.ts:454:22)
    at async getStaticProps (webpack-internal:///./pages/[filename].tsx:45:23)
    at async Object.renderToHTML (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\next\dist\server\render.js:390:20)
    at async doRender (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\next\dist\server\base-server.js:642:38)
    at async cacheEntry.responseCache.get.isManualRevalidate.isManualRevalidate (C:\Users\Max\Desktop\tina\my-tina-app\node_modules\next\dist\server\base-server.js:747:28)
    at async C:\Users\Max\Desktop\tina\my-tina-app\node_modules\next\dist\server\response-cache\index.js:80:36 {
  page: '/[filename]'
}
Terminate batch job (Y/N)? y
```

### Environment

Laptop, Windows 11

#### Tinacms Package Versions

```
C:\Users\Max\Desktop\tina\my-tina-app>npm ls
@tinacms/starter@0.1.0 C:\Users\Max\Desktop\tina\my-tina-app
+-- @headlessui/react@1.7.0
+-- @tailwindcss/typography@0.4.1
+-- @tinacms/cli@0.61.10
+-- @types/js-cookie@3.0.2
+-- @types/node@16.11.58
+-- @types/react@17.0.49
+-- @types/styled-components@5.1.26
+-- @typescript-eslint/eslint-plugin@5.36.2
+-- @typescript-eslint/parser@5.36.2
+-- autoprefixer@10.4.8
+-- date-fns@2.29.2
+-- eslint@8.23.0
+-- next-svgr@0.0.2
+-- next@12.3.0
+-- postcss-import@14.1.0
+-- postcss-nesting@10.1.10
+-- postcss@8.4.16
+-- react-dom@17.0.2
+-- react-icons@4.4.0
+-- react-is@17.0.2
+-- react@17.0.2
+-- styled-components@5.3.5
+-- styled-jsx@3.4.7
+-- tailwindcss@2.2.19
+-- tinacms@0.69.5
`-- typescript@4.8.3
```

#### Tools

| Tool    | Version |
| ------- | ------- |
| browser |      Chrome   |
| os      |       Windows 11  |
| node    |      16.15.0   |
| npm     |      8.5.5   |

**Comment by jamespohalloran** [comment]
I believe the underlying tinacms / Windows issue has been fixed, but each of the starters still need to be updated to the latest

**Comment by meel-hd** [comment]
I have the same exact error. I am using `tinacms:  "0.69.5"`

**Comment by BaseMax** [comment]
> I have the same exact error. I am using `tinacms: "0.69.5"`

Hi Mehdi, Thanks for reporting it, Would you please attach more details? @meel-hd

**Comment by meel-hd** [comment]
_@BaseMax_ 
OS:` Windows 11`
NodeJs `version 16.17.1`
yarn `v1.22.19`I first faced this problem with tina v0.69.5 but when I updated to v1.0.0 I still have the same error.
### Commad: `yarn dev`
```
yarn run v1.22.19
$ tinacms dev -c "next dev"

No format provided for collection page, defaulting to .md
        Typescript types => D:\-blog/.tina/__generated__/types.ts
        GraphQL types ====> D:\-blog/.tina/__generated__/schema.gql

Visit the GraphQL playground at http://localhost:4001/altair/
or
Enter the CMS at [your-development-url]/admin 

ready - started server on 0.0.0.0:3000, url: http://localhost:3000
event - compiled client and server successfully in 6.9s (1253 modules)
wait  - compiling /[filename] (client and server)...
event - compiled client and server successfully in 2.7s (1331 modules)
Unable to find collection for file at content\global\index.json

GraphQL request:10:3
 9 |     fragment LayoutQueryFragment on Query {
10 |   global(relativePath: "index.json") {
   |   ^
11 |     ...GlobalParts
More error context below
Unable to find collection for file at content\global\index.json
Error: Unable to find collection for file at content\global\index.json
    at TinaSchema.getCollectionByFullPath (D:\-blog\node_modules\@tinacms\graphql\dist\index.js:2676:15)
    at TinaSchema.getCollectionAndTemplateByFullPath (D:\-blog\node_modules\@tinacms\graphql\dist\index.js:2682:31)
    at Database.get (D:\-blog\node_modules\@tinacms\graphql\dist\index.js:3861:53)
    at async Resolver.getDocument (D:\-blog\node_modules\@tinacms\graphql\dist\index.js:3072:23)
    at async fieldResolver (D:\-blog\node_modules\@tinacms\graphql\dist\index.js:3705:39)
    at async Promise.all (index 0)
    at async Object.resolve (D:\-blog\node_modules\@tinacms\graphql\dist\index.js:3588:17)
    at async D:\-blog\node_modules\@tinacms\cli\dist\index.js:246:24 {
  locations: [ { line: 10, column: 3 } ],
  path: [ 'global' ]
}
error - Error: Unable to fetch, please see our FAQ for more information: https://tina.io/docs/errors/faq/

        Errors:
        Unable to find collection for file at content\global\index.json
    at TinaClient.request (D:\-blog\node_modules\tinacms\dist\client.js:74:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async requester (webpack-internal:///./.tina/__generated__/types.ts:454:22)
    at async getStaticProps (webpack-internal:///./pages/[filename].tsx:45:23)
    at async Object.renderToHTML (D:\-blog\node_modules\next\dist\server\render.js:390:20)
    at async doRender (D:\-blog\node_modules\next\dist\server\base-server.js:642:38)
    at async cacheEntry.responseCache.get.isManualRevalidate.isManualRevalidate (D:\-blog\node_modules\next\dist\server\base-server.js:747:28)     
    at async D:\-blog\node_modules\next\dist\server\response-cache\index.js:80:36 {
  page: '/[filename]'
}
wait  - compiling /_error (client and server)...
event - compiled client and server successfully in 851 ms (1332 modules)
```
![image](https://user-images.githubusercontent.com/92476321/202006697-4eeb63c5-8ea9-42ca-a63d-be7aae9e3f77.png)

**Comment by jamespohalloran** [comment]
@meel-hd Is your repo public by any chance? If so, would you mind sharing the repo link?

**Comment by meel-hd** [comment]
@jamespohalloran  I am afraid that is not possible The repo is private .

**Comment by Phoenix-Alpha** [comment]
@meel-hd , if you can't share your repo, can you provide as much detail as you can?
Did you use any starter template? If yes, which starter did you use?
Can you provide your `package.json` and schema file?

**Comment by meel-hd** [comment]
@Phoenix-Alpha 
### package.json
```
{
  "name": "@tinacms/starter",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "tinacms dev -c \"next dev\"",
    "build": "tinacms build && next build",
    "start": "tinacms build && next start",
    "export": "npm run build && next export",
    "lint": "eslint . --ext .ts,.tsx"
  },
  "devDependencies": {
    "@tinacms/cli": "0.61.10",
    "@types/js-cookie": "^3.0.0",
    "@types/node": "^16.11.7",
    "@types/react": "^17.0.35",
    "@types/styled-components": "^5.1.15",
    "@typescript-eslint/eslint-plugin": "^5.20.0",
    "@typescript-eslint/parser": "^5.20.0",
    "autoprefixer": "^10.4.0",
    "eslint": "^8.13.0",
    "postcss": "^8.3.11",
    "postcss-import": "^14.0.2",
    "postcss-nesting": "^10.1.0",
    "tailwindcss": "^2.2.19"
  },
  "dependencies": {
    "@headlessui/react": "^1.4.1",
    "@tailwindcss/typography": "^0.4.1",
    "date-fns": "^2.23.0",
    "next": "^12.0.4",
    "next-svgr": "^0.0.2",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-icons": "^4.2.0",
    "react-is": "^17.0.2",
    "styled-components": "^5.1.3",
    "styled-jsx": "^3.2.5",
    "tinacms": "0.69.5",
    "typescript": "^4.5.2"
  },
  "resolutions": {
    "final-form": "4.20.1",
    "@headlessui/react": "1.4.1"
  }
}

```
### schema
```
import { defineSchema, defineConfig, RouteMappingPlugin } from "tinacms";
import { contentBlockSchema } from "../components/blocks/content";
import { featureBlockSchema } from "../components/blocks/features";
import { heroBlockSchema } from "../components/blocks/hero";
import { testimonialBlockSchema } from "../components/blocks/testimonial";
import { iconSchema } from "../components/util/icon";
import { client } from "./__generated__/client";

const schema = defineSchema({
  config: {
    clientId: process.env.NEXT_PUBLIC_TINA_CLIENT_ID!,
    branch:
      process.env.NEXT_PUBLIC_TINA_BRANCH! || // custom branch env override
      process.env.NEXT_PUBLIC_VERCEL_GIT_COMMIT_REF! || // Vercel branch env
      process.env.HEAD!, // Netlify branch env
    token: process.env.TINA_TOKEN!,
    media: {
      // If you wanted cloudinary do this
      // loadCustomStore: async () => {
      //   const pack = await import("next-tinacms-cloudinary");
      //   return pack.TinaCloudCloudinaryMediaStore;
      // },
      // this is the config for the tina cloud media store
      tina: {
        publicFolder: "public",
        mediaRoot: "uploads",
      },
    },
  },
  collections: [
    {
      label: "Blog Posts",
      name: "post",
      path: "content/posts",
      format: "mdx",
      ui: {
        router: ({ document }) => {
          return `/post/${document._sys.filename}`;
        },
      },
      fields: [
        {
          type: "string",
          label: "Title",
          name: "title",
        },
        {
          type: "image",
          name: "heroImg",
          label: "Hero Image",
        },
        {
          type: "rich-text",
          label: "Excerpt",
          name: "excerpt",
        },
        {
          type: "reference",
          label: "Author",
          name: "author",
          collections: ["author"],
        },
        {
          type: "datetime",
          label: "Posted Date",
          name: "date",
          ui: {
            dateFormat: "MMMM DD YYYY",
            timeFormat: "hh:mm A",
          },
        },
        {
          type: "rich-text",
          label: "Body",
          name: "_body",
          templates: [
            {
              name: "DateTime",
              label: "Date & Time",
              inline: true,
              fields: [
                {
                  name: "format",
                  label: "Format",
                  type: "string",
                  options: ["utc", "iso", "local"],
                },
              ],
            },
            {
              name: "BlockQuote",
              label: "Block Quote",
              fields: [
                {
                  name: "children",
                  label: "Quote",
                  type: "rich-text",
                },
                {
                  name: "authorName",
                  label: "Author",
                  type: "string",
                },
              ],
            },
            {
              name: "NewsletterSignup",
              label: "Newsletter Sign Up",
              fields: [
                {
                  name: "children",
                  label: "CTA",
                  type: "rich-text",
                },
                {
                  name: "placeholder",
                  label: "Placeholder",
                  type: "string",
                },
                {
                  name: "buttonText",
                  label: "Button Text",
                  type: "string",
                },
                {
                  name: "disclaimer",
                  label: "Disclaimer",
                  type: "rich-text",
                },
              ],
              ui: {
                defaultItem: {
                  placeholder: "Enter your email",
                  buttonText: "Notify Me",
                },
              },
            },
          ],
          isBody: true,
        },
      ],
    },
    {
      label: "Global",
      name: "global",
      path: "content/global",
      format: "json",
      ui: {
        global: true,
      },
      fields: [
        {
          type: "object",
          label: "Header",
          name: "header",
          fields: [
            iconSchema,
            {
              type: "string",
              label: "Color",
              name: "color",
              options: [
                { label: "Default", value: "default" },
                { label: "Primary", value: "primary" },
              ],
            },
            {
              type: "object",
              label: "Nav Links",
              name: "nav",
              list: true,
              ui: {
                itemProps: (item) => {
                  return { label: item?.label };
                },
                defaultItem: {
                  href: "home",
                  label: "Home",
                },
              },
              fields: [
                {
                  type: "string",
                  label: "Link",
                  name: "href",
                },
                {
                  type: "string",
                  label: "Label",
                  name: "label",
                },
              ],
            },
          ],
        },
        {
          type: "object",
          label: "Footer",
          name: "footer",
          fields: [
            {
              type: "string",
              label: "Color",
              name: "color",
              options: [
                { label: "Default", value: "default" },
                { label: "Primary", value: "primary" },
              ],
            },
            {
              type: "object",
              label: "Social Links",
              name: "social",
              fields: [
                {
                  type: "string",
                  label: "Facebook",
                  name: "facebook",
                },
                {
                  type: "string",
                  label: "Twitter",
                  name: "twitter",
                },
                {
                  type: "string",
                  label: "Instagram",
                  name: "instagram",
                },
                {
                  type: "string",
                  label: "Github",
                  name: "github",
                },
              ],
            },
          ],
        },
        {
          type: "object",
          label: "Theme",
          name: "theme",
          fields: [
            {
              type: "string",
              label: "Primary Color",
              name: "color",
              options: [
                {
                  label: "Blue",
                  value: "blue",
                },
                {
                  label: "Teal",
                  value: "teal",
                },
                {
                  label: "Green",
                  value: "green",
                },
                {
                  label: "Red",
                  value: "red",
                },
                {
                  label: "Pink",
                  value: "pink",
                },
                {
                  label: "Purple",
                  value: "purple",
                },
                {
                  label: "Orange",
                  value: "orange",
                },
                {
                  label: "Yellow",
                  value: "yellow",
                },
              ],
            },
            {
              type: "string",
              name: "font",
              label: "Font Family",
              options: [
                {
                  label: "System Sans",
                  value: "sans",
                },
                {
                  label: "Nunito",
                  value: "nunito",
                },
                {
                  label: "Lato",
                  value: "lato",
                },
              ],
            },
            {
              type: "string",
              name: "icon",
              label: "Icon Set",
              options: [
                {
                  label: "Boxicons",
                  value: "boxicon",
                },
                {
                  label: "Heroicons",
                  value: "heroicon",
                },
              ],
            },
            {
              type: "string",
              name: "darkMode",
              label: "Dark Mode",
              options: [
                {
                  label: "System",
                  value: "system",
                },
                {
                  label: "Light",
                  value: "light",
                },
                {
                  label: "Dark",
                  value: "dark",
                },
              ],
            },
          ],
        },
      ],
    },
    {
      label: "Authors",
      name: "author",
      path: "content/authors",
      format: "md",
      fields: [
        {
          type: "string",
          label: "Name",
          name: "name",
        },
        {
          type: "string",
          label: "Avatar",
          name: "avatar",
        },
      ],
    },
    {
      label: "Pages",
      name: "page",
      path: "content/pages",
      ui: {
        router: ({ document }) => {
          if (document._sys.filename === "home") {
            return `/`;
          }
          if (document._sys.filename === "about") {
            return `/about`;
          }
          return undefined;
        },
      },
      fields: [
        {
          type: "object",
          list: true,
          name: "blocks",
          label: "Sections",
          ui: {
            visualSelector: true,
          },
          templates: [
            heroBlockSchema,
            featureBlockSchema,
            contentBlockSchema,
            testimonialBlockSchema,
          ],
        },
      ],
    },
  ],
});

export const tinaConfig = defineConfig({
  client,
  schema,
});

export default schema;
```

**Comment by Phoenix-Alpha** [comment]
@meel-hd , I could reproduce the issue. It was reported in #3171 before and has been fixed in #3175
I think this issue does not happen any more with latest package versions. Please confirm.
